gradle.properties

```diff
@@ -9,7 +9,7 @@ cordaProductVersion = 5.1.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 37
+cordaApiRevision = 38
```

Comment:
Can we either use a special Corda version, or an API revision way out of the range of the current 5.1 for this spike?
It will have to be updated to the then current 5.2 version anyway when/if merging to release.